### DISCUSSION TITLE
Upgrade - Failed with Pending Database Migrations

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#8052] [Q&A] [answered]
Encountered an issue today when upgrading from 0.15.4 to the latest release (running on Ubuntu 20.04.6 LTS)
Upon running:  apt upgrade inventree
The process went through the process, took maybe two minutes. After getting back to the command prompt, attempt to get back into the site.

I am able to get to the login page, enter the credentials and presented with a page with two notes:

`Pending Database Migrations
There are pending database migration which require attention. Contact your system administrator for further information`

Just below that, there is:
`Internal Server Error
The server raised an internal error
Refer to the error log in the admin interface for further details `

Not sure where to look or do to get back going again.

Appreciate any feedback....

**Comment by matmair** [comment, marked as answer]
Run `inventree run cli migrate`

**Reply by imager2019** [reply]
Thank you - that fixed it right up. 016.1 is running again. Quick testing shows all is good and can access data.

Appreciate the quick reply!
John
